### PR TITLE
Move frontend code to the root directory

### PR DESCRIPTION
**Pull request by nghialv**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: web/model/deployment_pb.js [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 var jspb = require('google-protobuf');
 var goog = jspb;
-var global = (function() {
+var global = (function () {
```
Comment (khanhtc1202) [inline web/model/deployment_pb.js:16]: could you run `yarn lint:fix` to recheck this kind of indent 👀 Sorry if you already did 🙏

Comment (nghialv) [reply web/model/deployment_pb.js:16]: Those files are generated code so let me fix them in another PR.
This PR should only be for the renaming of the directory.

Comment (nghialv) [reply web/model/deployment_pb.js:16]: Ah I got your point. Not sure why it was modified. Let me regenerate the code.

Comment (nghialv) [reply web/model/deployment_pb.js]: Addressed by https://github.com/pipe-cd/pipecd/pull/3376/commits/34f28eed158e8805636244aa564ec266ee7ace59

**Review by knanao — COMMENTED** [review]
Do you also change docs?
https://github.com/pipe-cd/pipecd/blob/master/docs/content/en/docs/contribution-guidelines/control-plane-development.md#source-code-structure

**Comment by khanhtc1202** [comment]
go bold
/lgtm

**Comment by nghialv** [comment]
> https://github.com/pipe-cd/pipecd/blob/master/docs/content/en/docs/contribution-guidelines/control-plane-development.md#source-code-structure

Nice catch! Thank you.
Let me update the docs in another PR.

**Comment by knanao** [comment]
Allright.
/approve

**Comment by nghialv** [comment]
Thank you.

**Pull request merged**